scripts/breaking_changes_checker/detect_breaking_changes.py

```diff
@@ -302,7 +302,9 @@ def test_compare_reports(pkg_dir: str, changelog: bool, source_report: str = "st
 
     remove_json_files(pkg_dir)
 
+    print("===== report changes begin =====")
```

Comment:
Nit: since this is going to show up for anyone running the tool, let's make the wording clearer:
```suggestion
    print("===== changelog start =====")
```